docs/docs/guides/how-to/cms-integration-plugin/index.mdx

```diff
@@ -0,0 +1,2053 @@
+    relations: ['translations'],
+  });
+  if (!product) return null;
+  const t = product.translations.find(tr => tr.languageCode === defaultLanguageCode);
+  if (!t?.slug) return null;
+  const doc = await this.findDocumentBySlug('vendure-product', t.slug);
+  return doc?.id ? String(doc.id) : null;
+}
+```
+
+---
+
+**CRUD Operations**
+
+These methods handle the basic Create, Read, Update, and Delete operations for documents in Payload. They follow REST API conventions and leverage the centralized request method for consistent behavior.
+
+```ts
+private async findDocumentBySlug(collection: string, slug: string) {
+  const res = await this.makePayloadRequest({ method: 'GET', endpoint: `${collection}?where[slug][equals]=${encodeURIComponent(slug)}&limit=1` });
+  return res.docs?.[0] ?? null;
+}
+
+private async createDocumentFromProduct(product: Product, defaultLanguageCode: LanguageCode, productSlug?: string | null) {
+  const data = await this.transformProductData(product, defaultLanguageCode, productSlug);
+  if (!data) return;
+  const res = await this.makePayloadRequest({ method: 'POST', endpoint: 'vendure-product', data });
+  Logger.info(`Created product ${product.id} (Payload ID: ${res.doc?.id})`);
+}
+
+// Additional CRUD methods like updateStoryFromProduct and deleteStoryFromProduct
+// follow similar patterns with PUT and DELETE HTTP methods respectively.
+// Full implementations are available in the working example repository.
+
+```
+
+**Configuration**
+
+Update your plugin configuration to include Payload options:
+
+```ts title="src/plugins/cms/types.ts"
+export interface PluginInitOptions {
+    // ... existing options
+    payloadApiKey?: string;
+    payloadBaseUrl?: string;
+}
+```
+
+**Environment Variables**
+
+Add these to your `.env` file:
+
+```env
+PAYLOAD_API_KEY=your_payload_api_key
+PAYLOAD_BASE_URL=http://localhost:3001/api
+```
+
+**Final Plugin Configuration**
+
+```ts title="src/vendure-config.ts"
+CmsPlugin.init({
+  payloadApiKey: process.env.PAYLOAD_API_KEY,
+  payloadBaseUrl: process.env.PAYLOAD_BASE_URL || 'http://localhost:3001/api',
+}),
+```
+
+This section provides a high-level overview. For a complete, production-ready implementation, see the Payload integration example:
+https://github.com/vendure-ecommerce/examples/tree/publish/examples/payload-cms-integration
+
+</TabItem>
+</Tabs>
+
+## Admin API Integration
+
+To allow for manual synchronization through graphQL mutations, we can extend the Admin API with new mutations. Let's use the CLI to generate the API extensions:
+
+```bash
+# Generate API extensions for the CMS plugin
+npx vendure add -a CmsSyncAdminResolver --selected-plugin CmsPlugin
+```
+
+### Extending the GraphQL API
+
+```ts title="src/plugins/cms/api/api-extensions.ts"
+export const adminApiExtensions = gql`
+  extend type Mutation {
+    syncProductToCms(productId: ID!): SyncResponse!
+    syncCollectionToCms(collectionId: ID!): SyncResponse!
+  }
+  // ...
+`;
```

Comment:
I think we should mention them also already right at the top of the guide, if someone is looking for a specific system.